src/app/search/directives/story-card.component.ts

```diff
@@ -18,7 +17,7 @@ import { StoryModel } from '../../shared';
       <section class="story-info">
         <span class="duration">{{storyDuration | duration}}</span>
         <span class="play-count"><i></i></span>
-        <span class="modified">{{storyDate | date:"MM/dd/yy"}}</span>
+        <span class="published">{{storyDate | date:"MM/dd/yy"}}</span>
```

Comment:
I don't see a published class related to this. Should remove class= duration and play-count too if getting rid of those.